dist/READMEpreview.md

```diff
@@ -0,0 +1,125 @@
+# OVN preview on OpenShift/origin/kubernetes
+
+WORK IN PROGRESS
+
+You can preview ovn overlay networks on an OpenShift cluster by replacing
+the installed SDN: Multitennant software with ovn components.
+
+NOTES:
+- This is limited to a single cluster master
+- This is limited to a single cluster network cidr range.
+- This is limited to OCP-3.10, OCP-3.11 (OCP-4.0 will install and work differently)
+
+## Installation:
+
+Install OCP-3.11 as instructed in the OCP documentation with the following change:
+1. In the ansible host file change
+```
+os_sdn_network_plugin_name='redhat/openshift-ovs-multitenant'
+```
+  to
+```
+os_sdn_network_plugin_name='cni'
+```
+  before running the cluster install playbook.
+
+When the cluster install is complete and the cluster is up there will
+be no networking. However, the openshift-sdn and openshift-ovs are installed
+and must be removed.
+
+```
+$ ssh root@master.node.com   <<<----- the cluster master node
+# oc project openshift-sdn
+# oc delete ds sdn
+# oc delete ds ovs
+```
+
+The needed ovn daemonsets and setup script are in the openvswitch/ovn-kubernetes
+repo on github.
+
+Clone ovn-kubernetes:
+```
+# git clone https://github.com/openvswitch/ovn-kubernetes
+# cd ovn-kubernetes/dist
+```
+
+Run the ovn-setup script to create the needed environment in the cluster:
+```
+# ./ansible/scripts/ovn-setup.sh
```

Comment:
There is an issue in ovn-setup.sh. The ClusterRoleBinding of 'cluster-admin-0' cannot be created. I pushed one PR to fix it 
https://github.com/openvswitch/ovn-kubernetes/pull/398